pkg/kubewarden/l10n/en-us.yaml

```
@@ -176,6 +176,15 @@ kubewarden:
     ignoreRancherNamespaces:
       label: Ignore Rancher Namespaces
       tooltip: Certain policies will break core services of Rancher, this will add a default list of namespaces to ignore.
+    namespaceSelector:
+      label: Namespace Selector
+      tooltip: NamespaceSelector decides whether to run the webhook on an object based on whether the namespace for that object matches the selector. If the object itself is a namespace, the matching is performed on object.metadata.labels. If the object is another cluster scoped resource, it never skips the webhook.
```

```suggestion
      tooltip: NamespaceSelector decides whether to evaluate the policy on a request, based on whether the namespace for that request matches the selector. If the object of the request is a namespace, the matching is performed on object.metadata.labels. If the object is another cluster scoped resource, it never skips the policy evaluation.
```

Trying to improve on the docs that come from https://doc.crds.dev/github.com/kubewarden/kubewarden-controller/policies.kubewarden.io/ClusterAdmissionPolicy/v1@v1.7.0-rc1#spec-namespaceSelector.

(Those docs get their doc string from the upstream namespaceSelector, which we don't want to touch for compatibility reasons. But that means they are not explicative if you don't know what a namespaceSelector is beforehand)